pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -174,18 +177,18 @@ func (c *Cluster) createSelfSignedCert() (bool, error) {
 	for i := 0; i < 5; i++ {
 		_, err := client.NewCephCommand(c.context, c.clusterInfo, args).RunWithTimeout(client.CmdExecuteTimeout)
 		if err == context.DeadlineExceeded {
-			logger.Infof("cert creation timed out. trying again..")
+			logger.Info("cert creation timed out. trying again..")
```

Comment:
Nit. Per golang syntax guidelines.

```suggestion
			logger.Info("cert creation timed out. trying again")
```

Reply:
Done.